samples/graph-app-installation-lifecycle/nodejs/Readme.md

```diff
@@ -0,0 +1,68 @@
+# App Installation
+
+This sample app demonstarte the installation lifecycle for Teams [Apps](https://docs.microsoft.com/en-us/graph/api/resources/teamsappinstallation?view=graph-rest-1.0) which includes create, update delete Apps
+
+
+## Prerequisites
+
+- [NodeJS](https://nodejs.org/en/)
+- [ngrok](https://ngrok.com/) or equivalent tunnelling solution
+
+## To try this sample
+
+1) Clone the repository
+
+    ```bash
+    git clone https://github.com/OfficeDev/Microsoft-Teams-Samples.git
+    ```
+
+1) In a terminal, navigate to `samples/graph-app-installation-lifecycle/nodejs
+`
+
+1) Install modules
+
+    ```bash
+    npm install
+    ```
+
+1) Run ngrok - point to port 3978
+
+    ```bash
+    ngrok http -host-header=rewrite 3978
+    ```
+
+
+1) Register your app with Microsoft identity platform via the Azure AD portal
+    - Your app must be registered in the Azure AD portal to integrate with the Microsoft identity platform and call Microsoft Graph APIs. See [Register an application with the Microsoft identity platform](https://docs.microsoft.com/en-us/graph/auth-register-app-v2).
+    - You need to add following permissions mentioned in the below screenshots to call respective Graph   API
+![](https://user-images.githubusercontent.com/50989436/116188975-e155a300-a745-11eb-9ce5-7f467007e243.png) 
+
+1) Update the `.env` configuration with the ```Microsoft App Id``` and ```App Password```
```

Comment:
ClientId & Client Secret.
You mentioned in .env file but in doc you are using "Micrisoft App Id" and  "App password"

Please be consistent here.